jped-cli/pom.xml

```diff
@@ -27,20 +27,20 @@
 			<version>1.3.1</version>
 		</dependency>
 		<dependency>
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
-			<version>4.11</version>
-			<scope>test</scope>
-		</dependency>
-		<dependency>
 			<groupId>com.github.samtools</groupId>
 			<artifactId>htsjdk</artifactId>
-			<version>1.128</version>
```

Comment:
Below it is broken

![selection_005](https://cloud.githubusercontent.com/assets/521780/11843281/10da7cd6-a409-11e5-9cf6-d624167b07e1.png)